projects/utils/src/lib/pipes/clean-array/clean-array.pipe.ts

```diff
@@ -0,0 +1,26 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+	name: 'cleanArray',
+})
+export class CleanArrayPipe implements PipeTransform {
+	/**
+	 * Removes all falsy values from the provided array.
+	 *
+	 * @param value The values that need to be stripped of falsy values.
+	 * @param exceptions The falsy values that may be included in the filtered array.
+	 * @returns The filtered array.
+	 */
+	public transform(value: any[], exceptions: any[] = []): any[] {
+		if (!exceptions.length) {
+			return value.filter(Boolean);
+		}
+
+		return value.reduce((acc, curr) => {
```

Comment:
Volgens mij kan je dit ook gewoon met een filter doen hoor, een reduce is hier niet echt nodig.